### ISSUE TITLE
Build broken

### ISSUE THREAD

**Issue by gicmo** [bendalab/NixView#87]
```
[ 80%] Building CXX object CMakeFiles/nix-view.dir/views/datatable.cpp.o
In file included from /Users/gicmo/Code/g-node/NixView/views/MainViewWidget.cpp:2:
In file included from /Users/gicmo/Code/g-node/NixView/build/ui_MainViewWidget.h:22:
In file included from /Users/gicmo/Code/g-node/NixView/views/projectnavigator.hpp:8:
/Users/gicmo/Code/g-node/NixView/db/projectindex.hpp:4:10: fatal error: 'QSql' file not found
#include <QSql>
```

```
-- Found Nix: /usr/local/lib/libnix.dylib
-- Boost version: 1.64.0
```

Qt installed via home-brew without modifications
```
qt: stable 5.8.0 (bottled), HEAD [keg-only]
```

Maybe one of the sql options need to be selected?
```
--with-mysql
	Build with mysql support
--with-postgresql
	Build with postgresql support
```

**Comment by gicmo** [comment]
This might have change more recently?
According to the [intro page](http://doc.qt.io/qt-5/qtsql-index.html) one shall include `QtSql` but then according to the [namespace page](http://doc.qt.io/qt-5/qsql.html) it is `QSql`.

**Comment by gicmo** [comment]
Fixed with #88
